contributor-docs/adrs/adr-013-primer-experimental-location.md

```diff
@@ -0,0 +1,62 @@
+# ADR 013: Location of experimental react components maintained by Primer
+
+## Status
+
+❌ Not Adopted, `<issue with implementation details goes here>`
+
+&nbsp;
+
+## Context: Experimental components (or drafts) in primer/react
+
+As we work on new components (or rewrites of old components), we often need to start with a "experiment" and not include it in the semantically versioned (semver-ed) main bundle.
+
+[ADR-006](./adr-006-drafts.md) explains our use of "drafts", here are the relevant parts:
+
+- When we are not sure about the API of a component, we put it in a different bundle: "primer/react/drafts". [ADR 010: Merge drafts status into experimental](./adr-010-drafts-are-experimental) recommends changing the name to `experimental`
+- The contract with consumers is that you are opting into a component in which we don't have 100% confidence yet, the API and structure can change.
+
+In [ADR 008: Lowering the barrier to entry](./adr-008-experimental-components) (Sep, 2022), we decided to create room to share experimental components between memex and the monolith with a new repository ([github/experimental-react-components](https://github.com/github/experimental-react-components)). Now that [memex is part of the monolith](https://github.com/github/planning-tracking/issues/1755), that [repository has been archived](https://github.com/github/platform-ux/issues/1248) (Feb, 2023).
+
+Which leaves us with: Where should experimental components maintained by Primer live?
+
+&nbsp;
+
+### Problems with publishing experimental components in @primer/react npm package:
+
+**Incompatible with semver**: Because we haven't reached the final API or structure, we need to ship breaking changes to experimental components. In the current setup, these breaking changes are part of minor or patch releases to primer/react.
+
+On the consumer side, this makes every upgrade a potentially breaking upgrade nullifying the benefits of semver.
+
+In the case of multiple teams working on the monolith, this can mean finding breaking changes for a new experimental component in a part of the app that you're not familiar with.
+
+&nbsp;
+
+## Decision
+
+1. Experimental components, as defined by the [component lifecycle](https://primer.style/contribute/component-lifecycle#experimental), should not be part of semantically versioned npm package for `@primer/react`.
+
+2. Each experimental component should be independently published and versioned with semver. Breaking changes should be tagged as major versions. Example: `@primer/react-experimental-underlinenav`.
```

Comment:
> Breaking changes should be tagged as major versions

_Post 1.0_. It's important to note that pre-1.0, it is totally acceptable and expected for components to have breaking changes. Once 1.0 has been reached, following breaking changes should cause a major version bump

Reply:
I can see that working as well. You can assume every release can be a breaking one, so major/minor might not be as informative or even as common.

If we follow pre-1.0 versions, I guess the ceremonious 1.0 would be a move to stable primer/react.